Copyright 2014 The Chromium Authors. All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are
met:

    * Redistributions of source code must retain the above copyright
notice, this list of conditions and the following disclaimer.
    * Redistributions in binary form must reproduce the above
copyright notice, this list of conditions and the following disclaimer
in the documentation and/or other materials provided with the
distribution.
    * Neither the name of Google Inc. nor the names of its
contributors may be used to endorse or promote products derived from
this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
ATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Date: Oct 31 08:24:49 2017

From: e9c25caa87cc5772aad1e254c46fda25047e1a38 Oct 31 08:24:49 2017

Subject: [PATCH] RDK-20604 : Fixing GCC_6.4 Wno-narrowing compilation error

	With the latest GCC_6.4 version some of the builds are failing
	in opencdm module for wnarrowing error. Hence, used the patch.

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: nhanasi <Naveenkumar_Hanasi@comcast.com>
---

diff --git a/src/browser/wpe/test/Makefile b/src/browser/wpe/test/Makefile
index d63ea05..1a56566 100644
--- a/src/browser/wpe/test/Makefile
+++ b/src/browser/wpe/test/Makefile
@@ -40,7 +40,7 @@ OCDM_INCLUDES = -I $(OCDM_ROOT_PATH) \
                 -I $(OCDM_WPE_ROOT_PATH)/opencdm \
                 $(OCDM_HEADER_PATH)
 
-CXFLAGS := -std=c++11 -g $(OCDM_INCLUDES) -pthread $(CXXFLAGS)
+CXFLAGS := -std=c++11 -g -Wno-narrowing $(OCDM_INCLUDES) -pthread $(CXXFLAGS)
 
 all: ocdm_client ocdm_decryptor
 	@echo "Building ocdm client"
